distributionviewer/core/static/js/app/components/views/chart-list.js

```diff
@@ -9,7 +9,7 @@ export class ChartList extends React.Component {
       <section className="chart-list">
         {this.props.items.map(chart => {
           return (
-            <ChartContainer key={chart.name} isDetail={false} chartName={chart.name} />
+            <ChartContainer key={chart.name} isDetail={false} chartId={chart.id} chartName={chart.name} />
```

Comment:
I'm not sure what these do tbh. I thought it was passing the values in some way. Are you saying we don't need `chartName` here then?